shell/src/main/java/alluxio/cli/fs/command/GetCmdStatusCommand.java

```diff
@@ -60,16 +60,11 @@ public String getDescription() {
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     long jobControlId = Long.parseLong(args[0]);
-
-    CmdStatusBlock cmdStatus = mClient.getCmdStatusDetailed(jobControlId);
-    if (cmdStatus.getJobStatusBlock().isEmpty()) {
-      System.out.format("Unable to get command status for jobControlId=%s, please retry.%n",
-              jobControlId);
-    } else {
-      System.out.format("Get command status information below: %n");
-      cmdStatus.getJobStatusBlock().forEach(block -> {
-        System.out.format("Job Id = %s, Status = %s %n", block.getJobId(), block.getStatus());
-      });
+    getDetailedCmdStatus(jobControlId);
```

Comment:
I remember you had a reason for putting GetCmdStatus under fs rather than job, could you explain again?

Reply:
Currently GetCmdStatus uses getDetailedCmdStatus in the AbstractDistributedJobCommand class, which would be a fs rather than job command.

Reply:
Can you review this updated PR @yuzhu